### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/Cast/CastBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        delete PostIdsStatuses.lastUserPost[currentAccount]
+
+      BurnerWalletStore.status = 'Posting cast'
+      const result = await PostStore.createPost(text)
+      const posts = await PostStore.posts
+      const numberOfPosts = await PostStore.postsAmount
+
+      PostStore.postsAmount = Promise.resolve(numberOfPosts + result.length)
+      for (const { id, post, derivativeAddress, sender, timestamp } of result) {
+        PostStore.posts = Promise.resolve([
```
Comment (T-Damer) [inline src/components/Cast/CastBlock.tsx:51]: This may cause double post bug, Sergey has removed it

File: src/components/Cast/CastBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        maxLength={maxLength}
+      />
+      {waitBurner ? (
+        <VerifyWallet
+          onCreateBurner={createPost}
+          status={status}
+          text={
+            BurnerWalletStore.status === 'Posting cast'
+              ? 'Hang tight! We’re casting now.'
+              : `
```
Comment (T-Damer) [inline src/components/Cast/CastBlock.tsx:105]: Let's use `''` not `

File: src/components/Cast/CastBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+        const blockchainId = id.toNumber()
+        const status = PostStatus.pending
+
+        PostIdsStatuses.lastUserPost = {
+          [currentAccount]: {
+            blockchainId,
+            status,
+          },
+          ...PostIdsStatuses.lastUserPost,
```
Comment (T-Damer) [inline src/components/Cast/CastBlock.tsx:70]: Hmm, what if the user already had `lastUserPost` and wrote another one from the same account, will the user have 2 same keys?

File: src/pages/Landing.tsx [hunk 4]
```diff
@@ -1,29 +1,37 @@
...
 export default function () {
   useScrollToTop()
 
   return (
-    <div className={blocksWrapper}>
-      <BurnerBlock />
-      <HowItWorks />
-      <BlockchainList />
+    <div className={processingCardWrapper}>
+      <>
```
Comment (T-Damer) [inline src/pages/Landing.tsx:25]: You don' need this `<>`

**Review by T-Damer — APPROVED** [review]
lgtm :)

File: src/components/Cast/CastBlock.tsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  const { status } = useSnapshot(BurnerWalletStore)
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState<unknown>()
+  const [text, setText] = useState('')
+  const [waitBurner, setWaitBurner] = useState(false)
+
+  const maxLength = 279
+  const errorMessage = error ? parseErrorText(error) : ''
+
+  async function createPost() {
```
Comment (smyzin) [inline src/components/Cast/CastBlock.tsx:27]: May we move it into the helpers dir?

File: src/components/Landing/ConnectAndCreateButton.tsx [hunk 6]
```diff
@@ -44,16 +24,9 @@ export default function ({
     try {
       if (!walletStore.account) await walletStore.connect(true)
       if (!walletStore.account) return onError('Please connect the wallet')
-      const hasFarcaster = await checkBadgeAndSignature(walletStore.account)
```
Comment (smyzin) [inline src/components/Landing/ConnectAndCreateButton.tsx:47]: We don't need this part anymore?

**Review by smyzin — APPROVED** [review]
Looks really good

File: src/components/ui/HorizontalDelimiter.tsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import classnames, { backgroundColor, height, width } from 'classnames/tailwind'
```
Comment (T-Damer) [inline src/components/ui/HorizontalDelimiter.tsx:1]: We need to remove this file

File: src/components/Footer/index.tsx [hunk 8]
```diff
@@ -48,7 +48,9 @@ export default function () {
               content={FooterOptions[key].text}
             />
             {linkList.length - 1 !== index && (
-              <Delimiter color="dimmed" showFrom="md" />
+              <div className={displayFrom('md')}>
+                <Delimiter />
```
Comment (T-Damer) [inline src/components/Footer/index.tsx:52]: imo dark blue-purple color has bad contrast:
image

File: src/index.css [hunk 9]
```diff
@@ -50,6 +52,12 @@
     var(--primary-dark) 90deg,
     var(--primary-background) 0
   );
+
+  --tertiary-subheader: linear-gradient(
+    270deg,
+    var(--primary-bright) 3.96%,
```
Comment (T-Damer) [inline src/index.css:58]: Why don't we use `4%` here?

File: src/icons/Thread.tsx [hunk 10]
```diff
@@ -0,0 +1,26 @@
+export default function () {
+  return (
+    <svg
+      width="24"
+      height="24"
+      viewBox="0 0 24 24"
+      fill="none"
```
Comment (T-Damer) [inline src/icons/Thread.tsx:7]: No need for this

File: src/components/ui/HorizontalDelimiter.tsx (hunk 7 above)
Comment (smyzin) [reply src/components/ui/HorizontalDelimiter.tsx:1]: Done

File: src/components/QuestionOfTheDay/StickLabel.tsx [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+  margin,
+  position,
+  zIndex,
+} from 'classnames/tailwind'
+
+const badgeWrapper = (mobile?: boolean) =>
+  classnames(
+    position('absolute'),
+    inset('right-0', mobile ? 'bottom-0' : 'top-0'),
+    margin(mobile ? '-mr-12.5' : '-mr-30', mobile ? '-mb-35' : '-mt-12.5'),
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/components/QuestionOfTheDay/StickLabel.tsx:13]: Fixed it

File: src/components/Footer/index.tsx (hunk 8 above)
Comment (smyzin) [reply src/components/Footer/index.tsx:52]: It looks like in the design. We may ask Matt on Monday about that

File: src/index.css (hunk 9 above)
Comment (smyzin) [reply src/index.css:58]: We don't use this class, so I removed it

File: src/icons/Thread.tsx (hunk 10 above)
Comment (smyzin) [reply src/icons/Thread.tsx:7]: Updated the whole icon

File: src/icons/Thread.tsx [hunk 12]
```diff
@@ -0,0 +1,26 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M16 9V8C16 6.89543 15.1046 6 14 6H4C2.89543 6 2 6.89543 2 8V14.7059C2 15.8105 2.89543 16.7059 4 16.7059H8.20685C8.71631 16.7059 9.20658 16.9003 9.57758 17.2495L11.1293 18.7099C11.8994 19.4347 13.1006 19.4347 13.8707 18.7099L15.3707 17.2981C15.7723 16.9202 16 16.3932 16 15.8417V15"
+        stroke="#EFECD6"
+        stroke-width="2"
+        stroke-linecap="round"
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/icons/Thread.tsx:14]: Removed it

File: src/hooks/usePost.ts [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+    let currentAccount
+    const { privateKey } = BurnerWalletStore
+    const { account } = walletStore
+
+    if (privateKey) currentAccount = new Wallet(privateKey).address
+    if (account && (await hasFarcasterBadge(account))) currentAccount = account
+
+    setError(null)
+    try {
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/hooks/usePost.ts:29]: Like this?

**Review by backmeupplz — APPROVED** [review]

**Pull request merged**